packages/e2e-test-utils-playwright/src/test.ts

```diff
@@ -146,6 +148,14 @@ const test = base.extend<
 		},
 		{ scope: 'worker' },
 	],
+	// See https://github.com/microsoft/playwright/issues/11058#issuecomment-999930304.
+	registerSelectors: [
+		async ( {}, use ) => {
+			await selectors.register( 'block', createBlockSelectorEngine );
+			await use();
+		},
+		{ scope: 'worker', auto: true },
+	],
```

Comment:
I guess we can just register it in `index.ts` though? That's how `role-selector` gets registered 😅.

Reply:
Oh yeah, probably!